requirements_dev.txt

```diff
@@ -17,3 +17,4 @@ vectormath
 wheel
 pytest
 git+https://github.com/simpeg/discretize.git@feat/3Dcylmesh
+git+https://github.com/simpeg/geoana@ref/em_and_docs
```

Comment:
I had to make couple of changes here, so take a look @lheagy

Reply:
thanks @sgkang: I would like to merge the geoana pr (simpeg/geoana#9) to master so that we can keep the requirements on the master geoana. For testing at the moment, this is fine though!